app/models/site_setting.rb

```diff
@@ -239,6 +243,10 @@ def self.create_style_settings site
       site.site_settings.find_or_initialize_by(name: 'footer-links-color', value: '\'white\'', position: 31)
     end
 
+    unless site.site_settings.exists?(name: 'header_login_enabled')
+      site.site_settings.find_or_initialize_by(name: 'header_login_enabled', value: 'false', position: 32)
```

Comment:
I think that by default, the login should be shown. Only a handful of projects will disable it.

Reply:
Sure, sounds fair. I'll wait for Jesus opinion, mine is that is just the way I wrote that condition, for making the changes.

Reply:
The faster solution is to change the condition but I think that the most consistent solution is to add a new migration or rake task which creates this new setting for all sites